lbz/events/api.py

```diff
@@ -46,6 +46,10 @@ def set_bus_name(self, bus_name: str) -> None:
     def sent_events(self) -> List[Event]:
         return deepcopy(self._sent_events)
 
+    @property
+    def bus_name(self) -> str:
```

Comment:
Also, it would be nice to declare it somewhere higher, not between `sent_events` and `pending_events` 😉